server.go

```diff
@@ -1903,6 +1903,40 @@ func (s *server) Start() error {
 			return nil
 		})
 
+		// If peers are specified as a config option, we'll add those
+		// peers first.
+		if len(s.cfg.AddPeers) != 0 {
```

Comment:
The `if` is not needed since `range` will not panic if a slice is nil or empty. That will also allow you to get rid of one level of indentation, making everything a bit less compressed.